### DISCUSSION TITLE
Github Mobile Bug: content of file vanishes when wrap lines option is on and switched between modes

### DISCUSSION THREAD

**Discussion by 10kartik** [community/community#30655] [Mobile] [answered]
I am using Github mobile v1.76.0 on an Andriod device. When I browse the source code of a repository in any mode(landscape/portrait), it displays the content perfectly but as soon as I switch mode (from landscape to portrait or portrait to landscape) the content vanishes, even after switching back to initial mode, the content doesn't load (isn't displayed). Only after navigating back and re-opening the file, the content is loaded. This bug is reproducible when 'wrap lines' mode is turned on.
Screenshot for reference: 
![Screenshot_20220824-171050](https://user-images.githubusercontent.com/99239411/186409682-4c08a1a7-bf18-4119-99a8-93d9ec12e028.jpg)
![Screenshot_20220824-170845](https://user-images.githubusercontent.com/99239411/186409735-f2ea2a2e-f254-46a9-98ad-1c8d1914a71b.jpg)

**Comment by aconsuegra** [comment, marked as answer]
Thanks for your feedback @10kartik! I have been able to reproduce this issue and I have created an internal issue to track it. I hope we can fix it soon!

**Reply by 10kartik** [reply]
Hello @aconsuegra, Thanks for your response. I was wondering if you could give me an idea what was the reason behind this bug's behaviour and how you guys resolved it. I mean, wherever the team manages to resolve it.. Maybe even after months! It'll be really helpful for my learning purpose. Thanks again!
